portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -71,15 +81,42 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String message = request.getParameter("message");
         String date = messageDate.toString();
 
+        //Prevents users from posting empty comments by checking the length of the username and message Strings:
         if(username.length() > 1 && message.length() > 1){
 
+            //Analise sentiment of the comment:
```

Comment:
This function is getting quite long. How about creating a helper function, something like `getSentimentScore` that does the following:

```
//Create a text document containing the message text:
Document commentDoc = Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();

//Intantiate a Language Service Client:
LanguageServiceClient languageService = LanguageServiceClient.create();

//Create a sentiment object with the analyzed text document:
Sentiment sentiment = languageService.analyzeSentiment(commentDoc).getDocumentSentiment();

//Return the sentiment score (Negative -1.0 to 1.0 Positive):
return sentiment.getScore();

```

Reply:
Changed it!